src/widgets/Circle.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
If we're doing this everywhere, should we just disable this in the linter config?

Reply:
I've suggested it but been overruled in the past!